test/tools/libtestutils/Common.cpp

```diff
@@ -52,11 +40,16 @@ boost::filesystem::path dev::test::getTestPath()
     return boost::filesystem::path(testPath);
 }
 
-int dev::test::randomNumber()
+int dev::test::randomNumber(int _min, int _max)
+{
+    g_randomGenerator.seed(random_device{}());
```

Comment:
Oh, this is not needed. Just do 
```
mt19937 g_randomGenerator(random_device{}());
```
once.

Reply:
Yea I was wondering why the old code seeded twice (once on creation and once per call)